server/pyproject.toml

```diff
@@ -36,6 +36,7 @@ channels-redis = "^3.1.0"
 pytest-asyncio = "^0.14.0"
 twisted = "^20.3.0"
 pytas = {git = "https://bitbucket.org/taccaci/pytas.git", tag = "v1.5.0"}
+gevent = "^20.9.0"
```

Comment:
may not matter, but is gevent a build or runtime dependency? Is it something needed to run efficiently in prod but not necessarily in dev?
https://www.python.org/dev/peps/pep-0518/#id26

Reply:
It's a runtime dependency, and only used in non-local configs ([locally runserver is used instead of uwsgi for auto-reloading](https://github.com/TACC/Frontera-Portal/blob/master/server/conf/docker/docker-compose-dev.all.debug.yml#L108)).

Unfortunately, however, I think poetry can only handle two dependency environments - `dependencies` and `dev-dependencies`. Opted to use `dev-dependencies` for local config stuff like test requirements that are not used in prod. So I'm not sure there's a better place to put `gevent` than in the `dependencies` list

Reply:
I see what you mean, at some point down the line this might be a good candidate for installation in Dockerfile instead. Ideally, we'd have a stand-alone django project that doesn't reference docker or any specific production runtime config at all. Those configs would be independent of the django project freeing users to adopt into whatever architecture works for them

This looks good though, we can work on where and how to break these out as we move to modularize the portal!